### ISSUE TITLE
Contour of qcriterion

### ISSUE THREAD

**Issue by WillTrojak** [Alpine-DAV/ascent#1133]
Hi, I want to produce a contour plot of q-criterion uing your qcriterion filter, but it doesn't seem possible. What I tried was a pipepline of the qcriterion filter taking velocity as an input with an output name of `q` and then in the pipepline I have a contour filter that consumes 'q'. I then have a scene that uses this pipeline. Below is te error message that's given:

```
file: /Users/wtrojak/Documents/code/pyfr_ascent_venv/ascent/src/libs/ascent/runtimes/ascent_main_runtime.cpp
line: 1909
message: 
Execution failed with vtkm: Could not find appropriate cast for array in CastAndCall.
Array: valueType=vtkm::Vec<vtkm::Vec<float, 3>, 3> storageType=vtkm::cont::StorageTagBasic 43614208 values occupying 1570111488 bytes [((0.999937,2.42981e-07,0), ...) ... (... (6.10085e-07,-2.3063e-07,0))]
TypeList: vtkm::List<float, double, vtkm::Vec<float, 2>, vtkm::Vec<double, 2>, vtkm::Vec<float, 3>, vtkm::Vec<double, 3>, vtkm::Vec<float, 4>, vtkm::Vec<double, 4> >
```

**Comment by cyrush** [comment]
@WillTrojak Thanks for the report, I can reproduce this in a unit test and I will look into the root cause.

**Comment by cyrush** [comment]
@WillTrojak  I believe I understand the root of this issue and I am converging on a fix.

**Comment by WillTrojak** [comment]
Great, thanks for looking into this! We have acsent support in PyFR now and q-criterion is something people really like to plot.

**Comment by cyrush** [comment]
@WillTrojak this was resolved in #1138 which been merged to develop.
